index.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en" ng-app="ToDoList">
+  <head>
+    <meta charset="utf-8">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="bower_components/bootstrap/dist/css/bootstrap.css">
+    <script src="bower_components/jquery/dist/jquery.js"></script>
+    <script src="bower_components/angular/angular.js"></script>
+    <script src="bower_components/angular-resource/angular-resource.js"></script>
+    <script src="bower_components/angular-xeditable/dist/js/xeditable.js"></script>
+    <script src="js/app.js"></script>
+    <script src="js/tasksController.js"></script>
+    <script src="js/navigationController.js"></script>
+  </head>
+
+  <body ng-controller="TasksController as controller">
```

Comment:
maybe taskCtrl? Otherwise when you add another controller it gets confusing

That's the point of the `Something as something` syntax - for clarity